src/components/recipe-form.js

```diff
@@ -136,13 +138,41 @@ const RecipeForm = ({
   }
 
   const handleSubmit = async values => {
+    if (name != values.name) {
+      log += 'Name, '
+    }
+    if (JSON.stringify(ingredients) != JSON.stringify(values.ingredients)) {
+      log += 'Ingredients, '
+    }
+    if (instructions != values.instructions) {
+      log += 'Instructions, '
+    }
+    if (prep_time != values.prep_time) {
+      log += 'Prep Time, '
+    }
+    if (cook_time != values.cook_time) {
+      log += 'Cook Time, '
+    }
+    if (servings != values.servings) {
+      log += 'Servings, '
+    }
+    if (image_url != values.image_url) {
+      log += 'Image, '
+    }
+
+    //remove trailing ', ' if present
+    if (log != '') {
+      log = log.substring(0, log.length - 2)
+    }
+
```

Comment:
One thought on this. Do we want to save the log as a big string or as an array? Either way, if `log` was an array at first, we could `push` all of the values into `log` and then just do a `.join(',')` at some point to turn it into a string with commas in between. Then we don't have to remove the trailing comma. Just a thought

Reply:
yeet